definitions/npm/validate-currency-code_v0.x.x/flow_v0.104.x-/validate-currency-code_v0.x.x.js

```diff
@@ -0,0 +1,20 @@
+declare module 'validate-currency-code' {
+  declare module.exports: string => boolean;
+}
+
+declare module 'validate-currency-code/codes' {
+  declare module.exports: string[];
+}
```

Comment:
Doesn't looks like it's really public, or at least not intended to be public. I would remove it.